tox.ini

```diff
@@ -19,12 +20,28 @@ commands = python setup.py build_sphinx
 commands = oslo-config-generator --config-file=etc/armada/config-generator.conf
 
 [testenv:lint]
-commands = flake8 .
+commands = flake8
+ignore=E302,H306,E402,W503
```

Comment:
i know that we used this test, but we should figure out a better way to approach this